grpc/src/state_change_proxy.rs

```diff
@@ -220,7 +220,7 @@ mod tests {
         state_change_interface::{StateChangeCommand, StateChangeInterface},
         test_utils::{generate_test_complete_state, generate_test_workload_spec_with_param},
     };
-    use tokio::sync::mpsc;
+    use tokio::{join, sync::mpsc};
```

Comment:
This is the other place where we have a misalignment with main and a commit gets reverted.